added brazilian portuguese translation for exchange and teamwork PSA (#110)
monero-ecosystem/outreach-docs: monero-outreach-docs/translations/pt-br/PSA/files/exchange/exchange_pt-br.md

# Conheça a segurança da sua corretora

> Corretoras podem ser uma maneira conveniente de vender e comprar Monero, mas nem todas elas foram criadas da mesma forma.

Reviewer: ```suggestion
As corretoras podem ser uma maneira conveniente de vender e comprar Monero, mas nem todas elas foram criadas da mesma forma.
```

> ### Aqui estão algumas dicas para você ter em mente quando se trata de guardar seus XMR de forma segura nas corretoras.

Reviewer: ```suggestion
### Confira a seguir algumas dicas para você guardar seus XMR com segurança em corretoras.
```

> 1. Faça a sua própria pesquisa antes de abrir conta em qualquer corretora. Tente responder essas perguntas:

Reviewer 2: ```suggestion
1. Faça a sua própria pesquisa antes de abrir uma conta em qualquer corretora. Tente responder essas perguntas:
```

> 1. Faça a sua própria pesquisa antes de abrir uma conta em qualquer corretora. Tente responder essas perguntas:

Reviewer: ```suggestion
1. Sempre faça a sua própria pesquisa antes de abrir uma conta em qualquer corretora. Ou seja, informe-se por conta própria, não confie apenas em recomendações de outras pessoas. Ao analisar a segurança de uma corretora, tente responder as seguintes perguntas:
```

- a. Existem avaliações positivas de outras pessoas que a utilizam? Essas avaliações são confiáveis?

> - b. Essa corretora já foi atacada? O combate ao ataque foi apropriado?

Reviewer: ```suggestion
- b. Essa corretora já sofreu algum ataque hacker? As medidas para combater o ataque foram apropriadas?
```

> - c. A quanto tempo a corretora está operando?

Reviewer: ```suggestion
- c. Há quanto tempo a corretora está em operação?
```

> - d. Os criadores/administradores são conhecidos ou são indivíduos ou entidades de reputação?

Reviewer 2: ```suggestion
- d. Os criadores/administradores são conhecidos ou são indivíduos ou entidades com boa reputação?
```

> - d. Os criadores/administradores são conhecidos ou são indivíduos ou entidades com boa reputação?

Reviewer: ```suggestion
- d. Os seus fundadores e/ou administradores são conhecidos na comunidade? Eles são indivíduos ou entidades com boa reputação?
```

> - e. É possível de manter sua privacidade, não fornecendo nenhuma informação pessoal?

Reviewer 2: ```suggestion
- e. É possível manter sua privacidade, não fornecendo nenhuma informação pessoal?
```

> - f. O website utiliza precauções de segurança como captcha ou autenticação de dois fatores (2FA)? Ele possui certificados de segurança legítimos e começa com https://?

Reviewer 2: ```suggestion
- f. O website utiliza precauções de segurança como Captcha ou autenticação de dois fatores (2FA)? Ele possui certificados de segurança legítimos e começa com https://?
```

> - g. Se você planeja declarar impostos, a corretora permite que você faça o download de comprovantes de transações de compra/venda?

Reviewer 2: ```suggestion
- g. Se você planeja declarar impostos, a corretora permite que você faça o download de comprovantes de transações de compra e venda?
```

> 2. As corretoras criam carteiras próprias delas para você. Você dificilmente terá acesso a semente mnemônica da carteira da corretora. Se a corretora for atacada ou representar um golpe, você poderá perder todos os XMR que estão na carteira da corretora. Uma boa prática é apenas guardar XMR em corretoras se você planeja utilizá-los em breve. Caso contrário, transfira para uma carteira da qual você tenha controle.

Reviewer 2: ```suggestion
2. As corretoras criam carteiras próprias delas para você. Você dificilmente terá acesso à semente mnemônica da carteira da corretora. Se a corretora for atacada ou tentar dar um golpe, você poderá perder todos os XMR que estão na carteira da corretora. Uma boa prática é apenas guardar XMR em corretoras se você planeja utilizá-los em breve. Caso contrário, transfira para uma carteira na qual você tenha controle.
```

> 2. As corretoras criam carteiras próprias delas para você. Você dificilmente terá acesso a semente mnemônica da carteira da corretora. Se a corretora for atacada ou tentar dar um golpe, você poderá perder todos os XMR que estão na carteira da corretora. Uma boa prática é apenas guardar XMR em corretoras se você planeja utilizá-los em breve. Caso contrário, transfira para uma carteira da qual você tenha controle.

Reviewer: ```suggestion
2. Embora as corretoras criem carteiras de Monero para você, você não terá controle total sobre os fundos dessas carteiras, pois as corretoras não costumam fornecem acesso à semente mnemônica que gerou a carteira. Se a corretora sofrer um ataque hacker ou tentar dar um golpe, você poderá perder todos os XMR que estão na carteira da corretora. Uma boa prática é guardar em corretoras apenas os XMR que você planeja usar em breve. Se você quer guardar alguns XMR a longo prazo, o recomendado é que você os transfira para uma carteira que você tenha total controle (uma carteira que você tenha a semente mnemônica).
```

> 3. Se você quiser trocar uma quantidade significativa de XMR por meio de uma corretora, considere fazer um teste primeiro, com uma transação de valor mais baixo. Esse valor deve ser baixo o suficiente para que você não se sinta mal no caso de perdê-lo.

Reviewer 2: ```suggestion
3. Se você quiser trocar uma quantidade significativa de XMR por meio de uma corretora, considere fazer um teste primeiro, com uma transação de valor mais baixo. Esse valor deve ser baixo o suficiente para que você não se sinta mal caso venha a perdê-lo.
```

4. Confira sempre, em qualquer transação de XMR, os endereços das carteiras que vão enviar/receber Monero para ter certeza de que estão corretos. Mesmo que você tenha copiado e colado os endereços, confira se você copiou o endereço correto, pois existem ataques que modificam endereços de criptomoedas quando estes são copiados e colados. Se o seu aparelho estiver infectado por esse vírus, uma simples dupla verificação para confirmar o endereço copiado será o suficiente para proteger seus XMR.

> 5. Com o passar do tempo, as corretoras descentralizadas serão mais comuns. Como o Monero é uma criptomoeda descentralizada, essas corretoras estão de acordo com o caráter do Monero. Considere utilizá-las, mas como sempre, faça sua pesquisa primeiro!

Reviewer 2: ```suggestion
5. Com o passar do tempo, as corretoras descentralizadas serão mais comuns. Como o Monero é uma criptomoeda descentralizada, essas corretoras estão de acordo com o espírito do Monero. Considere utilizá-las, mas como sempre, faça sua pesquisa primeiro!
```
